src/sentry/models/project.py

```diff
@@ -98,6 +98,7 @@ class Project(Model):
         'pk',
         'slug',
     ])
+    platform = models.CharField(max_length=64, null=True)
```

Comment:
you might need to coordinate with @getsentry/ops on this one, as im not sure locks on project are ok/not ok at this point